src/pages/components/inline.mdx

```diff
@@ -71,6 +71,8 @@ import SizeCode from '!!raw-loader!../../examples/loaders/inline/InlineSize.tsx'
 
 <Component components={props.data.mdx.components} componentName="inline" />
 
+Nest an Inline within a messaging context to signal activity of another party.
```

Comment:
This is a bit vague... we may need a more concrete example of usage.

Reply:
If we're asking questions like this, then I think it might be a sign that this is more design guidance documentation and not API documentation. Because this content here reminds me of Lucijan's work on the loaders pattern documentation: https://docs.google.com/document/d/1tkeY3LGm8yGuNdiIzY1LuhJnNKmdyQN9Zrt3ZvRPvWw/edit

Reply:
Good point, yeah I blurred the line here between design guidance and API docs.. good to know for the future, and thanks for sharing.